osu.Game/Overlays/Settings/Sections/Input/KeyBindingRow.cs

```diff
@@ -530,6 +534,9 @@ public void UpdateKeyCombination(KeyCombination newCombination)
                 if (KeyBinding.RulesetName != null && !RealmKeyBindingStore.CheckValidForGameplay(newCombination))
                     return;
 
+                if (RulesetBindings.Select(k => k.KeyCombination).Contains(newCombination))
+                    return;
```

Comment:
So when a user inputs the same key in another action, it'll just silently ignore it? There should be visual feedback, at least flashing red on the key binding button.

Reply:
Is flashing red enough?

Reply:
Is the color `Color4.Red` enough?

Reply:
Sounds fine.

Reply:
It looks something like this with `Color4.Red` as the color of the box.

![Screenshot from 2022-09-04 20-42-30](https://user-images.githubusercontent.com/54083498/188314086-619058a8-2fd0-4cc2-8882-6812c1d5da53.png)